README.adoc

```diff
@@ -0,0 +1,67 @@
+= New Bonita Documentation Site
+// External URIs:
+:url-antora: https://antora.org
+:url-asciidoctor: https://asciidoctor.org/
+:url-node: https://nodejs.org
+:url-nvm: https://github.com/creationix/nvm
+:url-nvm-install: {url-nvm}#installation
+
+
+*_DISCLAIMER: this is a work in progress._*
+
+
+== Architecture
+
+Static site generation based on {url-antora}[Antora] using {url-asciidoctor}:[Asciidoctor] documentation content stored in various Git repositories.
+
+
+== Build
+
+=== Requirements
+
+
+{url-node}[Node 12.x] (may work with newer versions, no guarantee). For more details, see the  https://docs.antora.org/antora/2.3/install-and-run-quickstart/[Antora installation guide].
+
+While you can install Node from the official packages, we strongly recommend that you use {url-nvm}[nvm] (Node Version Manager) to install and manage Node.
+Follow the {url-nvm-install}[nvm installation instructions] to set up nvm on your machine.
+
+=== Build the static site for production
+
+Run
+```bash
+npm install
+npm run build
+```
+The static site is then available in the `build/site` folder.
+
+
+=== Local development
+
+The `antora` CLI is available by calling `./node_modules/.bin/antora --version`
+
+For more CLI options, see https://docs.antora.org/antora/2.3/cli/options/
+
+
+==== Building for local developments
+
+For a site that can be load from local disk
+```bash
+npm run local:build-static
+```
+
+For a site to be served by the development <<http server>>
+```bash
+npm run local:build
+```
+
+
+==== http server
+
+Once the static site is build, you can run `npm run serve` and access to http://localhost:8080/ to check if everything is working correctly (urls in taxononmy, links, ....).
```

Comment:
This would require to change the script in the `package.json` file as well. I suggest we start with the script names I proposed and then review them all with the whole team.
I don't care of keeping of changing the names, I just don't want to ping pong rename after each dev review (as we all have different taste in terms of naming)